package.json

```diff
@@ -6,6 +6,7 @@
   "author": "Christian Johansen",
   "license": "BSD-3-Clause",
   "main": "./lib/samsam",
+  "types": "./dist/samsam/samsam.d.ts",
```

Comment:
Tested, and works as expected 🎉

<img width="749" alt="Screenshot 2020-11-12 at 12 37 40" src="https://user-images.githubusercontent.com/6163988/98935512-ee5a3f00-24e3-11eb-93bd-d8751789b6af.png">

I had to change the types path though, it should be `./dist/samsam.d.ts`

Reply:
I've fixed that in a new commit... I think.

Would you mind verifying again?

Reply:
LGTM, it works 🚀